src/api/tenantApi.js

```diff
@@ -0,0 +1,116 @@
+import axios from 'axios'
+const BASE_URL = 'https://dev.elevate-apis.shikshalokam.org'
+
+// Get all tenants
+export const getAllTenants = async () => {
+  const response = await axios.get(`${BASE_URL}/user/v1/tenant/list`, {
```

Comment:
all api urls (eg /user/v1/tenant/list) should be in a single constant file so that its easy to modify at one place.